### DISCUSSION TITLE
How to correctly identify used storage space?

### DISCUSSION THREAD

**Discussion by kuznetsov-as** [minio/minio#20750] [Q&A]
Hello!

I've got 4 nodes with 1 disks each, each disk got 89Gb available. I am using docker for my deployment.

EC:2

I am trying to understand the size of all the files in my cluster, to do this I first used df -h:

```
          Size  Used  Avail  Use%  Mounted on
minio-01 = 89G   16G    69G   19%  /opt/minio
minio-02 = 89G   35G    50G   42%  /opt/minio
minio-03 = 89G   43G    42G   51%  /opt/minio
minio-04 = 89G   25G    60G   30%  /opt/minio
```

Amount of space used: 119G

I also tried to see the used space with the command:

mc du myminio
29GiB   102431 objects

I also called the following command:

mc admin heal --json -r myminio | tail -n 1
{"status":"success","type":"summary","objects_scanned":102804,"objects_healed":546,"items_scanned":102950,"items_healed":546,"size":31561681084,"duration":1538}

Can you please answer why the df command displays so much used space? Why is the data unevenly distributed across the disks?
What's the difference between objects_scanned and items_scanned? Am I correct in understanding that with EC:2, the total amount of space used on all disks should be around 60G?

**Comment by klauspost** [comment]
If everything is running as it should the usage would be exactly the same.

Items are all types, objects are just objects:
```
	HealItemMetadata       HealItemType = "metadata"
	HealItemBucket                      = "bucket"
	HealItemBucketMetadata              = "bucket-metadata"
	HealItemObject                      = "object"
```

**Reply by kuznetsov-as** [reply]
@klauspost please tell me why there is a big difference in the number of objects in the output of the command  **mc du myminio** (102431 objects) and **mc admin heal --json -r myminio | tail -n 1** ("objects_scanned":102804)?

**Reply by klauspost** [reply]
I don't remember the details of these two pieces of code.
